common/static/common/js/spec/components/paging_spec.js

```diff
@@ -58,9 +58,7 @@ define([ "jquery", "js/common_helpers/ajax_helpers", "URI",
             initialize : function() {
                 this.registerSortableColumn('name-col', 'Name', 'name', 'asc');
                 this.registerSortableColumn('date-col', 'Date', 'date', 'desc');
-                this.registerFilterableColumn('js-asset-type-col', gettext('Type'), 'asset_type');
```

Comment:
Note to reviewer: the filterable columns are not tested, so I didn't see any reason to register them ("asset_type" is no longer a valid column name anyway).